ifrs17/Import/Importers.ipynb

```diff
@@ -248,10 +248,10 @@
         {
             "cell_type": "code",
             "source": [
-                "async public Task CleanDatabaseAsync<T> (Expression<Func<T, bool>> filter = null) where T : class",
+                "public static async Task CleanIfrsTargetAsync<T> (this IDataSource dataSource, Expression<Func<T, bool>> filter = null) where T : class",
```

Comment:
I suggest: CleanIfrsTargetAsync --> CleanAsync
Reasons: concise is better, "Ifrs" is confusing since the method is generic for every type, the added value of "Target" is negligible (IMO)